autoconfigure/collector-kinesis/README.md

```diff
@@ -0,0 +1,61 @@
+# autoconfigure-collector-kinesis
+
+## ZipkinKinesisCollectorAutoConfiguration
+
+This is a Spring Boot [AutoConfiguration](http://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-auto-configuration.html) 
+module that can be added to a [Zipkin Server](https://github.com/openzipkin/zipkin/tree/master/zipkin-server) 
+deployment to collect Spans from Amazon Kinesis streams.  Internally
+this module wraps the [KinesisCollector](https://github.com/openzipkin/zipkin-aws/tree/master/collector-kinesis) 
+and exposes configuration options through environment variables.
+
+## Usage
```

Comment:
you can probably hijack aka copy/paste find/replace this https://github.com/openzipkin/zipkin/pull/1586/files#diff-9400707464efbebd3dd6b1e4821c5f79R8